scripts/deblur

```diff
@@ -453,13 +453,11 @@ def build_biom_table(seqs_fp, output_biom_fp, min_reads, file_type, log_level, l
               show_default=True,
               help="indicates this is not the main deblur process "
               "(used by the parallel deblur - do not use manually)")
-@click.option('attack_mode', '-A', required=False, type=bool, default=False,
-              help="ATTACK MODE (http://xkcd.com/1692)")
```

Comment:
Every OTU picking method needs an Attack mode!!!!
(would prefer to keep it :) )